users/config.rst

```diff
@@ -373,20 +377,20 @@ blockPullOrder
     Available options:
 
     standard (default):
-        The file is split into N equal continuous part sequences, where N is the number of connected
-        devices, each device starts downloading it's own sequence, after which it picks other devices
-        sequence at random. Provides acceptable data distribution, minimal spinning disk strain.
+        The blocks of a file are split into N equal continuous sequences, where N is the number of connected
```

Comment:
There is no term blocks mentioned before, so I think wedging a new term half way is confusing.